src/components/scenes/CreateWalletSelectCryptoScene.tsx

```diff
@@ -237,7 +232,14 @@ const CreateWalletSelectCryptoComponent = (props: Props) => {
     () => (
       <Fade noFadeIn={defaultSelection.length > 0} visible={numSelected > 0} duration={300}>
         <View style={styles.bottomButton}>
-          <MainButton label={lstrings.string_next_capitalized} type="primary" marginRem={[0, -0.5]} onPress={handleNextPress} alignSelf="center" />
+          <MainButton
+            label={lstrings.string_next_capitalized}
+            type="primary"
+            marginRem={[0, -0.5]}
+            onPress={handleNextPress}
+            alignSelf="center"
+            disabled={numSelected === 0}
```

Comment:
The next button is disabled by the Fade component so this doesn't do anything

Reply:
The fade animation isn't enough, you can still click it while fading. This change was to remove the disabled check in handleNextPress

Reply:
You can replicate it easily by having the last wallet to disable be an entry near the next button

Reply:
Checked this again, can't reproduce. I can just drop this commit

Reply:
Do you have a video of the behavior? I still can't replicate and from code inspection it doesn't seem possible. The Fade component disables pointer events when the component is hidden and in this case that's when selected is 0. You may still be able to see it but pointer events have turned off.